### ISSUE TITLE
>> for match in regex.search_all("One  Two \n\tThree"): << example code produces an error

### ISSUE THREAD

**Issue by golddotasksquestions** [godotengine/godot-docs#4586]
**Your Godot version:** 3.2.3

**Issue description:**
This current example code does not work and produces an error:
```
Example of splitting a string using a RegEx:

var regex = RegEx.new()
regex.compile("\\S+") # Negated whitespace character class.
var results = []
for match in regex.search_all("One  Two \n\tThree"):
    results.push_back(match.get_string())
# The `results` array now contains "One", "Two", "Three".
```
This works fine:

```
for i in regex.search_all("One  Two \n\tThree"):
    results.push_back(i.get_string())
```

**URL to the documentation page:**
https://docs.godotengine.org/en/3.2/classes/class_regex.html#class-regex-method-search

**Comment by Calinou** [comment]
This was fixed by https://github.com/godotengine/godot/pull/45075, but the online documentation wasn't updated yet.

This will be fixed once the class reference in the `3.2` branch is synchronized.

**Comment by madmiraal** [comment]
Fixed in https://github.com/godotengine/godot/pull/45075 and included in 3.3:
https://docs.godotengine.org/en/3.3/classes/class_regex.html
